### PR TITLE
Feature/smhe 1050 add timezone on import shipment file

### PR DESCRIPTION
**Pull request by harrymiddelburgetall**

### CODE REVIEW

**Comment by harrymiddelburgetall** [comment]
❓ Nightly: https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/554/

**Comment by harrymiddelburgetall** [comment]
✔ Nightly: https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/565/

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/ad-hoc/SynchronizeTime.feature [hunk 1]
```diff
@@ -12,3 +12,6 @@ Feature: SmartMetering AdHoc
       | DeviceIdentification | TEST1024000000001 |
     Then the date and time is synchronized on the device
       | DeviceIdentification | TEST1024000000001 |
+    Then the dlms device with identification "TEST1024000000001" exists with properties
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/ad-hoc/SynchronizeTime.feature]: And instead of Then

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AddSmartMeter.feature [hunk 2]
```diff
@@ -21,12 +21,15 @@ Feature: SmartMetering Installation - Add smart meter
       | Encryption_key        | SECURITY_KEY_E    |
       | ManufacturerCode      | Test              |
       | ModelCode             | Test              |
+      | Timezone              | Europe/Amsterdam  |
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AddSmartMeter.feature]: Also add a test without timezone

File: osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/SynchronizeTimeCommandExecutorTest.java [hunk 3]
```diff
@@ -91,6 +96,10 @@ void testSynchronizeTime() throws ProtocolAdapterException, IOException {
     assertThat(resultCode).isEqualTo(AccessResultCode.SUCCESS);
     verify(this.dlmsConnection, times(1)).set(this.setParameterArgumentCaptor.capture());
 
+    // save timezone
```
Comment (kroesctrl) [inline osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/SynchronizeTimeCommandExecutorTest.java]: Add test for null value timezone

**Review by kroesctrl — CHANGES_REQUESTED** [review]

File: osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/SynchronizeTimeCommandExecutorTest.java (hunk 3 above)
Comment (harrymiddelburgetall) [reply osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/SynchronizeTimeCommandExecutorTest.java]: Timezone is required in SynchronizeTimeRequest. No need to test it here
`@XmlElement(name = "TimeZone", required = true)`

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AddSmartMeter.feature (hunk 2 above)
Comment (harrymiddelburgetall) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AddSmartMeter.feature]: Added test for E-meter without Timezone. Added check for empty timezone in AddMbusDevice.feature

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/ad-hoc/SynchronizeTime.feature (hunk 1 above)
Comment (harrymiddelburgetall) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/ad-hoc/SynchronizeTime.feature]: Then -> And

**Comment by harrymiddelburgetall** [comment]
✔ Nightly: https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-apps-smart-metering-integration-nightly-build/308/

**Review by kroesctrl — APPROVED** [review]

**Comment by harrymiddelburgetall** [comment]
✔ Nightly: https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/577/
